source/robot_model/test/fixtures/ur5e.urdf

```diff
@@ -172,7 +172,7 @@
     <visual>
       <origin rpy="0 0 3.141592653589793" xyz="0 0 0"/>
       <geometry>
-        <mesh filename="package://ur_description/meshes/ur5e/visual/base.dae"/>
+        <mesh filename="/src/source/robot_model/test/fixtures/ur5e/visual/base.dae"/>
```

Comment:
We should do something to not forget to remove these hardcoded paths at some point. either a FIXME in the code or an issue

Reply:
Here is an issue `https://github.com/aica-technology/control-libraries/issues/165` that aim to solve this at a later date.